source/comply/audit-log.rst

```diff
@@ -1,7 +1,7 @@
 Audit logging (beta)
 ====================
 
-.. include:: ../_static/badges/allplans-selfhosted.rst
+.. include:: ../_static/badges/ent-cloud-selfhosted.rst
```

Comment:
I believe that audit logging (beta) is available only on self-hosted deployments, and not something a Cloud admin can work with.